.github/workflows/coverage_test.yaml

```diff
@@ -1,10 +1,13 @@
 name: coverage-test
 on: [pull_request, push]
+env:
+  CC: gcc-9
+  CXX: g++-9
 
 jobs:
     build:
         name: coverage-test
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-20.04
```

Comment:
what s the lcov version with 20 ? 1.14.2 ?

Reply:
at least on my install, yes